### DISCUSSION TITLE
Update on scroll when using safePolygon?

### DISCUSSION THREAD

**Discussion by AgainstLightning** [floating-ui/floating-ui#1692] [Q&A] [answered]
First, thank you for the killer work done on this project! You're doing awesome stuff here.

I have a question: My ToolTip handles close with `safePolygon` as my tooltip includes a link. However now when I have moused over and activated a ToolTip but then scroll my viewport, without moving my mouse, the tooltip remains open while my cursor leaves the tooltip reference element. Removing `safePolygon` fixes this.

I am assuming this has to do with it being updated on `pointermove` as noted in the docs. 

How do I address this? How do I benefit from safePolygon's functionality without losing the ability to scroll as expected?

Thank you!

**Comment by atomiks** [comment, marked as answer]
Sounds like it should listen for `mouseleave` somewhere in addition to the `move`

Adding `ancestorScroll: true` to `useDismiss()` is one workaround for that problem now.

**Reply by AgainstLightning** [reply]
Setting `ancestorScroll: true` in `useDismiss` closes on scroll but does so regardless of the cursor being over the reference element or not. This is not the desired behaviour either. 🤔

**Reply by atomiks** [reply]
Gonna try to make a PR for this

**Reply by atomiks** [reply]
Fixed in `v0.6.0`

**Reply by AgainstLightning** [reply]
Awesome, you rock! Thank you for what you do!
